packages/react-instantsearch/src/connectors/connectBreadcrumb.js

```diff
@@ -0,0 +1,134 @@
+import PropTypes from 'prop-types';
+import createConnector from '../core/createConnector';
+import { refineValue, getResults } from '../core/indexUtils';
+
+export const getId = props => props.attributes[0];
+
+const namespace = 'hierarchicalMenu';
+
+function refine(props, searchState, nextRefinement, context) {
+  const id = getId(props);
+  const nextValue = { [id]: nextRefinement || '' };
+  const resetPage = true;
+  return refineValue(searchState, nextValue, context, resetPage, namespace);
+}
+
+function transformValue(values) {
+  return values.reduce((acc, item) => {
+    if (item.isRefined) {
+      acc.push({
+        label: item.name,
+        // If dealing with a nested "items", "value" is equal to the previous value concatenated with the current label
+        // If dealing with the first level, "value" is equal to the current label
+        value: item.path,
+      });
+      // Create a variable in order to keep the same acc for the recursion, otherwise "reduce" returns a new one
+      if (item.data) {
+        acc = acc.concat(transformValue(item.data, acc));
+      }
+    }
+    return acc;
+  }, []);
+}
+
+/**
+ * The breadcrumb component is s a type of secondary navigation scheme that 
+ * reveals the user’s location in a website or web application.
+ * 
+ * If you want to use the "defaultRefinement" prop, you can do so by using a Virtual Hierarchical Menu 
```

Comment:
I think this should be rephrase a bit. 
The breadcrumb doesn't have any `defaultRefinement` prop, so this might be confusing. 

Maybe have something like => "If you want to select a specific refinement for your Breadcrumb you'll need to use a Virtual Hierarchical Menu..." And then present a usage example. 

This should be present also on the widget guide and maybe only on this one.